docs/api-docs/getting-started/deprecations-and-sunsets.md

```diff
@@ -0,0 +1,58 @@
+# Deprecations and Sunsets
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Deprecations](#deprecations)
+- [Sunsets](#sunsets)
+- [Related resources](#related-resources)
+
+</div>
+
+## Deprecations
+
+The following endpoints are deprecated. We discourage using these endpoints as they are no longer supported by BigCommerce. Instead, consider using the provided V3 replacements.
```

Comment:
The following endpoints are deprecated. We discourage using these endpoints as BigCommerce no longer supports them. Instead, consider using the provided V3 replacements.